cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -70,11 +70,14 @@ func ValidateNodeConfiguration(c *kubeadm.NodeConfiguration) field.ErrorList {
 	return allErrs
 }
 
-func ValidateAuthorizationMode(authzMode string, fldPath *field.Path) field.ErrorList {
-	if !authzmodes.IsValidAuthorizationMode(authzMode) {
-		return field.ErrorList{field.Invalid(fldPath, authzMode, "invalid authorization mode")}
+func ValidateAuthorizationModes(authzModes []string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	for _, authzMode := range authzModes {
```

Comment:
We should make sure here that you can't set it to `RBAC,RBAC,ABAC,ABAC` or the like. Duplicate values aren't allowed